docs-new/docs/graphql/core/remote-schemas/auth/remote-schema-permissions.mdx

```diff
@@ -0,0 +1,349 @@
+---
+description: Remote schema permissions
+keywords:
+  - authorization
+  - docs
+  - remote schema
+  - permissions
+sidebar_position: 3
+---
+
+import Thumbnail from "@site/src/components/Thumbnail/Thumbnail";
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Remote schema permissions{#remote-schema-permissions}
+
+## Introduction
+
+Hasura supports [role-based authorization](/graphql/core/auth/authorization/index.mdx) for remote schemas.
+
+Remote schema permissions can be defined to:
+
+1. Expose only certain parts of the remote schema to a role
+2. Preset arguments with static values or session variables for any field.
+
+:::info Supported from
+
+Remote schema permissions are supported in Hasura GraphQL engine
+versions `v2.0.0` and above.
+
+:::
+
+:::info Enable remote schema permissions
+
+Remote schema permissions are **not** enabled by default.
+
+To enable them, you will have to run the GraphQL engine either with the server flag `--enable-remote-schema-permissions` or environment variable `HASURA_GRAPHQL_ENABLE_REMOTE_SCHEMA_PERMISSIONS` set to `true`.
+
+When remote schema permissions are not enabled, all remote schemas are considered to be public entities, i.e. all roles will have unrestricted access to the remote schemas. Once enabled, access to remote schemas will be restricted for all roles (other than `admin`) unless explicit permissions for the remote schema fields are granted to the roles.
+
+:::
+
+## Role based remote schemas
```

Comment:
missing custom id - {#role-based-remote-schemas}
```suggestion
## Role based remote schemas {#role-based-remote-schemas}
```

Please have the original `rst` file open always to avoid similar misses.

Reply:
Ack